src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -1,165 +1,137 @@
-import { initialPlotConfigStates } from '../../../../../../redux/reducers/componentConfig/initialState';
-import generateEmbeddingCategoricalSpec from '../../../../../../utils/plotSpecs/generateEmbeddingCategoricalSpec';
-import generateEmbeddingContinuousSpec from '../../../../../../utils/plotSpecs/generateEmbeddingContinuousSpec';
-import colorProvider from '../../../../../../utils/colorProvider';
+
+import isBrowser from '../../../../../../utils/environment';
 import DimensionsRangeEditor from '../../../plots-and-tables/components/DimensionsRangeEditor';
 import ColourbarDesign from '../../../plots-and-tables/components/ColourbarDesign';
 import ColourInversion from '../../../plots-and-tables/components/ColourInversion';
-import LogExpression from '../../../plots-and-tables/embedding-continuous/components/LogExpression';
 import AxesDesign from '../../../plots-and-tables/components/AxesDesign';
 import PointDesign from '../../../plots-and-tables/components/PointDesign';
 import TitleDesign from '../../../plots-and-tables/components/TitleDesign';
 import FontDesign from '../../../plots-and-tables/components/FontDesign';
 import LegendEditor from '../../../plots-and-tables/components/LegendEditor';
 import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
+import { filterCells } from '../../../../../../utils/plotSpecs/generateEmbeddingCategoricalSpec';
+import { loadCellSets } from '../../../../../../redux/actions/cellSets';
 
 const { Panel } = Collapse;
 
 const EmbeddingPreview = () => {
-  const dispatch = useDispatch();
-
   const router = useRouter();
   const { experimentId } = router.query;
-  const [selectedSpec, setSelectedSpec] = useState('sample');
-  const [plotSpec, setPlotSpec] = useState({});
-  const [config, setConfig] = useState(null);
-
-  const embeddingConfig = useSelector((state) => state.experimentSettings.processing);
-  const embedding = useSelector((state) => state.embeddings);
-  const embeddingMethod = embeddingConfig?.configureEmbedding?.embeddingSettings?.method;
+  const [selectedPlot, setSelectedPlot] = useState('sample');
+  const [plot, setPlot] = useState(false);
+  const cellSets = useSelector((state) => state.cellSets);
 
-  const error = false;
+  const dispatch = useDispatch();
 
   const plots = {
     sample: {
-      title: 'Samples',
-      initialConfig: initialPlotConfigStates.embeddingCategorical,
-      specGenerator: generateEmbeddingCategoricalSpec,
+      title: 'Colored by Samples',
       imgSrc: plot1Pic,
+      plotUuid: 'embeddingPreviewBySample',
+      plotType: 'embeddingPreviewBySample',
     },
     cellCluster: {
-      title: 'Default clusters',
-      initialConfig: initialPlotConfigStates.embeddingCategorical,
-      specGenerator: generateEmbeddingCategoricalSpec,
+      title: 'Colored by CellSets',
       imgSrc: plot1Pic,
+      plotUuid: 'embeddingPreviewByCellSets',
+      plotType: 'embeddingPreviewByCellSets',
```

Comment:
isn't `plotType` equal to either `embeddingContinuous` or `embeddingCtegorical` in this `plots` object?

Reply:
Will there be difference in settings between these plots and `embeddingCategorial` and `embeddingContinuous`? If not then I'll change them back,